src/test/runner.ts

```diff
@@ -140,14 +140,23 @@ export class Runner {
     const testFileFilter = testFileReFilters.length ? createMatcher(testFileReFilters.map(e => e.re)) : () => true;
     const config = this._loader.fullConfig();
 
+    let projectsToFind: Map<string, string> | undefined;
+    if (projectName) {
+      projectsToFind = new Map();
+      projectName.split(',').forEach(n => {
+        const name = n.trim();
+        projectsToFind!.set(name.toLocaleLowerCase(), name);
+      });
+    }
     const projects = this._loader.projects().filter(project => {
-      return !projectName || project.config.name.toLocaleLowerCase() === projectName.toLocaleLowerCase();
+      return !projectsToFind || projectsToFind.delete(project.config.name.toLocaleLowerCase());
```

Comment:
This breaks when you have multiple projects with the same name. Our tests do that :)

Reply:
This should be address by #8604 which prohibits duplicate project names.